[Workers for Platforms] Fix getting started guide (#10070)
cloudflare/cloudflare-docs: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md

You will now upload `customer-worker-1` into your dispatch namespace that you created in step 2. This user Worker has a simple `fetch()` handler that sends a `Hello world` response.

```js
---
filename: main.js
---
export default {
  fetch(request) {
    return new Response('Hello World');
  },
};
```

> Moreover, you need to define a simple metadata file for the user Worker:

Reviewer: ```suggestion
Do the following to define a simple metadata file for the user Worker:
```
